src/Core/CustomRedirects/DefaultRedirectsService.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using BVNetwork.NotFound.Core.Data;
+
+namespace BVNetwork.NotFound.Core.CustomRedirects
+{
+    public class DefaultRedirectsService : IRedirectsService
+    {
+        private readonly IRepository<CustomRedirect> _repository;
+        private readonly IRedirectLoader _redirectLoader;
+
+        public DefaultRedirectsService(
+            IRepository<CustomRedirect> repository, IRedirectLoader redirectLoader)
+        {
+            _repository = repository ?? throw new ArgumentNullException(nameof(repository));
+            _redirectLoader = redirectLoader ?? throw new ArgumentNullException(nameof(redirectLoader));
+        }
+
+        public IEnumerable<CustomRedirect> GetAll()
+        {
+            return _redirectLoader.GetAll();
+        }
+
+        public IEnumerable<CustomRedirect> GetAllExcludingIgnored()
+        {
+            return _redirectLoader.GetByState(RedirectState.Saved);
```

Comment:
this does not include suggestion

Reply:
so maybe better name is needed

Reply:
Yes, this is only for redirects, not for suggestions.